tornjak-frontend/src/components/entry-create.tsx

```diff
@@ -619,164 +637,279 @@ class CreateEntry extends Component<CreateEntryProp, CreateEntryState> {
-                    helperText="Ttl for identities issued for this entry (In seconds)"
-                    id="ttl-input"
-                    invalidText="Number is not valid"
-                    label="Time to Leave (Ttl)"
-                    min={0}
-                    step={1}
-                    value={0}
-                    onChange={this.onChangeTtl}
-                  />
+        <br /><br />
+        {this.state.message !== "" &&
+          <div>
+            <ToastNotification className="toast-entry-creation-notification"
+              kind="info"
+              iconDescription="close notification"
+              subtitle={
+                <span>
+                  <br></br>
+                  <div role="alert" data-test="success-message">
+                    {this.state.statusOK === "Multiple" &&
+                      <div>
+                        <p className="success-message">{"-- " + this.state.successNumEntries.success + " ENTRY/ ENTRIES SUCCESSFULLY CREATED--"}</p>
+                        {this.state.successNumEntries.fail !== 0 &&
+                          <p className="failed-message">{"-- " + this.state.successNumEntries.fail + " ENTRY/ ENTRIES FAILED TO BE CREATED--"}</p>
+                        }
+                      </div>
+                    }
+                    {this.state.statusOK === "OK" && this.state.successJsonMessege === "OK" &&
+                      <p className="success-message">--ENTRY SUCCESSFULLY CREATED--</p>
+                    }
+                    {(this.state.statusOK === "ERROR" || (this.state.successJsonMessege !== "OK" && this.state.successJsonMessege !== "")) &&
+                      <p className="failed-message">--ALL ENTRIES CREATION FAILED--</p>
+                    }
+                  </div>
+                  <br></br>
+                  <div className="toast-messege" data-test="alert-primary">
+                    <pre className="toast-messege-color">
+                      {this.state.message}
+                    </pre>
+                  </div>
+                </span>}
+              timeout={0}
+              title="Entry Creation Notification"
+            />
+            {window.scrollTo({ top: 0, behavior: 'smooth' })}
+          </div>
+        }
+        <Accordion className="accordion-entry-form">
+          <AccordionItem
+            title={<h5>Upload New Entry/ Entries</h5>} open>
+            <div className="entry-form">
+              <CreateEntryYaml
+                ParentIdList={ParentIdList_noManualOption} />
+              <br></br>
+              {this.props.globalNewEntries.length === 0 &&
+                <div>
+                  <Button
+                    size="medium"
+                    color="primary"
+                    variant="contained"
+                    disabled
+                    >
+                    Create Entries
+                  </Button>
+                  <p style={{ fontSize: 13 }}>i.e. Upload File to Enable</p>
```

Comment:
Thank you for disabling the button when no file uploaded. 
Please change the text from: 
`i.e. Upload File to Enable` to 
`(Upload JSON File to Enable)`

Reply:
done